Week5/csvwrite.py

```diff
@@ -0,0 +1,7 @@
+import csv
+f=open('c:\\sqlite3\\csv\\student.csv','w',newline='')
```

Comment:
Is there any other method to open file...?

Reply:
in open()- you have to close the file
in with open()-file close automatically without you telling it